roles/deploy_bmh/template/bmh-nmstate.yml.j2

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ node_name }}-nmstate-secret
+  namespace: {{ cimfw_deploy_bmh_namespace }}
```

Comment:
AFAIK, this should be in `openshift-machine-api`

Reply:
We are passing that value in the downstream file I have, but I'll change the default to a more sane one.
In any case, and just for your information, it seems it's possible to watch all namespaces by setting `watchAllNamespaces: true` in the ` Provisioning` CR.